version.gradle

```diff
@@ -1,3 +1,3 @@
 ext {
-  PROSPECTIVE_RELEASE_VERSION = '2.0.1'
+  PROSPECTIVE_RELEASE_VERSION = '2.0.2'
```

Comment:
Fcuk, I shouldn't have merged PR #917... YOLO, pls keep the version at 2.0.1, _wir schaffen es_ :trollface: (FYI @mkondratek)

Reply:
Undone on #924

Reply:
I mean, version bump

Reply:
I have to change it after #924 merge